go/vt/vttablet/tabletserver/throttle/throttler.go

```diff
@@ -84,7 +84,7 @@ func registerThrottlerFlags(fs *pflag.FlagSet) {
 
 	fs.DurationVar(&throttleThreshold, "throttle_threshold", throttleThreshold, "Replication lag threshold for default lag throttling")
 	fs.StringVar(&throttleMetricQuery, "throttle_metrics_query", throttleMetricQuery, "Override default heartbeat/lag metric. Use either `SELECT` (must return single row, single value) or `SHOW GLOBAL ... LIKE ...` queries. Set -throttle_metrics_threshold respectively.")
-	fs.Float64Var(&throttleMetricThreshold, "throttle_metrics_threshold", throttleMetricThreshold, "Override default throttle threshold, respective to -throttle_metrics_query")
+	fs.Float64Var(&throttleMetricThreshold, "throttle_metrics_threshold", throttleMetricThreshold, "Override default throttle threshold, respective to --throttle_metrics_query")
```

Comment:
not actually relevant to the changes in this PR - what does `respective to --throttle_metrics_query` mean?
maybe a question more for @shlomi-noach

Reply:
It comes hand-in-hand with `--throttle_metrics_query`. If you define a `--throttle_metrics_query`, then you should also say what's the `--throttle_metrics_threshold` at which the throttle will engage. Not sure how to phrase it otherwise.